recipes/libmysqlclient/all/conanfile.py

```diff
@@ -1,23 +1,45 @@
+from conan.tools.microsoft import msvc_runtime_flag
+from conan.tools.files import rename
 from conans import ConanFile, CMake, tools
 from conans.errors import ConanInvalidConfiguration
-from conans.tools import Version
+import functools
 import os
 
+required_conan_version = ">=1.36.0"
+
 
 class LibMysqlClientCConan(ConanFile):
     name = "libmysqlclient"
     url = "https://github.com/conan-io/conan-center-index"
     description = "A MySQL client library for C development."
-    topics = ("conan", "mysql", "sql", "connector", "database")
+    topics = ("mysql", "sql", "connector", "database")
     homepage = "https://dev.mysql.com/downloads/mysql/"
     license = "GPL-2.0"
-    exports_sources = ["CMakeLists.txt", "patches/*"]
-    generators = "cmake"
+
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False], "with_ssl": [True, False], "with_zlib": [True, False]}
-    default_options = {"shared": False, "fPIC": True, "with_ssl": True, "with_zlib": True}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_ssl": [True, False],
+        "with_zlib": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_ssl": True,
+        "with_zlib": True,
+    }
+
+    short_paths = True
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "pkg_config"
```

Reply:
`pkgconf` should be added to `build_requirements()`.

```cmake
IF(UNIX)
  IF(FREEBSD)
    MYSQL_CHECK_PKGCONFIG()
    PKG_CHECK_MODULES(LIBUNWIND libunwind)
```